index.html

```diff
@@ -207,119 +203,113 @@ <h2 data-i18n>Realtime Sync</h2>
             </div>
           </section>
           <section class="paper">
-            <h2 data-i18n>Settings</h2>
+            <h2 data-i18n>Settings</h2>          
```

Comment:
There is some whitespace at the end of this line. Could you install prettier extension and format-on-save to make sure, the format is not changed?